sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.azure.messaging.eventhubs.checkpointstore.redis;
```

Comment:
A couple minor things here:

- All files we author must contain a Microsoft licensing header, generally it is the following:

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

I'm not sure which IDE you're using, but if it is IntelliJ there are configurations for `File and Code Templates` where you can configure all new files to automatically include this header.

- Package name is indicative of the file structure, so in this case I'd have expected this file to be contained in `/src/com/azure/messaging/eventhubs/checkpointstore/redis`. I'd recommend creating that as it'll be the base point for all sub-packages such as `com.azure.messaging.eventhubs.checkpointstore.redis`. If you're using IntelliJ there should be a right click option to add a package to a source folder, I always do each individually instead of one `com.azure.messaging.eventhubs.checkpointstore.redis` as it sometimes will create a single folder with that name which is different than subfolders but shows up the same in the UI.

Reply:
Thank you, this helped!